src/models/history/WirewayAssembly.ts

```diff
@@ -29,20 +30,12 @@ const le = 0
 const he = 1
 const wi = 2
 const pd = 3
-const pa = 4
 
 export const paramsMap: Param[] = [
   { index: le, name: 'Length', type: ParamType.Slider, value: 200, step: 5, values: [100, 400] },
-  { index: he, name: 'Height', type: ParamType.Slider, value: 40, step: 5, values: [20, 80] },
-  { index: wi, name: 'Width', type: ParamType.Slider, value: 60, step: 5, values: [20, 120] },
+  { index: he, name: 'Height', type: ParamType.Slider, value: 30, step: 5, values: [20, 80] },
+  { index: wi, name: 'Width', type: ParamType.Slider, value: 30, step: 5, values: [20, 120] },
   { index: pd, name: 'Position', type: ParamType.Slider, value: 0, step: 5, values: [0, 100] },
-  {
```

Comment:
Didn't work as expected with changing the product, so I decided to remove it completely and let the example only change by single parameters.